multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -419,6 +419,13 @@ def __init__(
         self._model_loaded = False  # Whether the model has been loaded
         self._matcher = None
 
+        if not self._save_path:
+            self._save_path = setup_outputdir(
+                path=None,
+                warn_if_exist=self._warn_if_exist,
+            )
+        self._save_path = os.path.abspath(os.path.expanduser(self._save_path))
```

Comment:
I think this can cause issues for `save_path`. If users specify `path` in init, predictor would create a new path in fit.